src/target/riscv/riscv.c

```diff
@@ -5029,6 +5040,73 @@ int riscv_dmi_write_u64_bits(struct target *target)
 	return r->dmi_write_u64_bits(target);
 }
 
+static int check_if_trigger_is_availible(struct target *target, unsigned int t)
```

Comment:
There is a typo in the original name.

You can also rename it to the following (which is easier to read and equally clear):

```suggestion
static int check_if_trigger_exists(struct target *target, unsigned int t)
```

Reply:
Addressed.